SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialRPCService.h

```diff
@@ -110,7 +127,18 @@ class SPATIALGDK_API SpatialRPCService
 	TMap<EntityComponentId, Schema_ComponentData*> PendingRPCsOnEntityCreation;
 
 	TMap<EntityComponentId, Schema_ComponentUpdate*> PendingComponentUpdatesToSend;
-	TMap<EntityRPCType, TArray<RPCPayload>> OverflowedRPCs;
+	TMap<EntityRPCType, TArray<RPCPayload>> QueuedRPCs;
+
+	EPushRPCResult FailureOutcomesToQueue;
 };
 
+namespace PushRPCResultUtils
+{
+	bool IsAnyFlagSet(const EPushRPCResult Result, const EPushRPCResult Flags);
+	EPushRPCResult GetFailureAction(const EPushRPCResult Result);
```

Comment:
Am I missing something, since I'm not sure we need these functions? Isn't it the case that `EPushRPCResult` should only have 1 value in it?

Reply:
No, EPushRPCResult is now storing a BitMask, similar to EObjectFlags.